### DISCUSSION TITLE
It doesn't work in my vs-code

### DISCUSSION THREAD

**Discussion by GarasChan** [community/community#11153] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/22443356/152909402-6a1f695b-6c30-429b-ad7e-16374cc8c5e6.png)

**Comment by GarasChan** [comment]
I am not sure that if the 'auto close tags' extension stopped it.

**Comment by davecheney** [comment]
Copilot is working, please try a different example or open a program that you are working on.

**Comment by KTibow** [comment]
Your cursor seems to be on your bracket. Make sure your cursor is in the empty space, and if nothing shows up, try Alt+\ or Ctrl+Enter.
